pkg/itemblock/actions/pvc_action.go

```diff
@@ -105,9 +105,67 @@ func (a *PVCAction) GetRelatedItems(item runtime.Unstructured, backup *v1.Backup
 		}
 	}
 
+	// Gather groupedPVCs based on VGS label provided in the backup
+	groupedPVCs, err := a.getGroupedPVCs(context.Background(), pvc, backup)
+	if err != nil {
+		return nil, err
+	}
+
+	// Add the groupedPVCs to relatedItems so that they processed in a single item block
+	relatedItems = append(relatedItems, groupedPVCs...)
+
 	return relatedItems, nil
 }
 
 func (a *PVCAction) Name() string {
-	return "PodItemBlockAction"
+	return "PVCItemBlockAction"
+}
+
+// getGroupedPVCs returns other PVCs in the same group based on the VGS label key in the Backup spec.
+func (a *PVCAction) getGroupedPVCs(ctx context.Context, pvc *corev1api.PersistentVolumeClaim, backup *v1.Backup) ([]velero.ResourceIdentifier, error) {
+	var related []velero.ResourceIdentifier
+
+	vgsLabelKey := backup.Spec.VolumeGroupSnapshotLabelKey
+	if vgsLabelKey == "" {
+		a.log.Info("No VolumeGroupSnapshotLabelKey provided in backup spec; skipping PVC grouping")
```

Comment:
Should this be a debug log? I'm not sure we need a new log msg for every PVC backed up here if we're not using VGS.

Reply:
Also, this should basically never trigger, right? Since velero will fill in the default one if one isn't specified? Still need the check, but it shouldn't actually get to this line if things are working right.

Reply:
Done, updated the PR !